Nesteo.Server/Controllers/Api/InspectionsController.cs

```diff
@@ -164,5 +165,18 @@ public async Task<ActionResult<InspectionPreview>> GetInspectionPreviewByIdAsync
 
             return inspectionPreview;
         }
+
+        /// <summary>
+        /// Export all inspections
+        /// </summary>
+        [HttpPatch("export")]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        public IAsyncEnumerable<string> ExportInspectionsAsync()
```

Comment:
`[HttpGet("csv")]` would be a better fit for this download-function I think.
Please change the name/summary to something in the sense of "Download CSV" then.
Status code is 200 OK.

Same for the nesting boxes please.